### ISSUE TITLE
iPhone 13 pro returning wrong dpi?

### ISSUE THREAD

**Issue by leossmith** [airsdk/Adobe-Runtime-Support#2023]
## Problem Description

Describe your problem in detail. Include the following information:

AIR SDK 33.1.1.889
iOS 15.x / iPhone 13 pro

So I had a weird issue with one of our apps made with Flex and latest AIR SDK. It seems that the runtimeDPI returned when running the app is 163dpi for iPhone which of course is not true. That leads to everything being pixelated. Anyone had the same issue? I dont remember this being a problem in earlier devices.

**Comment by raresn** [comment]
@ajwfrost this happens for iphone 16 pro as well - DeviceCapabilities.dpi reports 401 as dpi. 
and @marchbold 's Application Ane reports correct (460).

**Comment by marchbold** [comment]
These are generally done from look up tables so likely something needs to be updated here for the newer devices.

**Comment by raresn** [comment]
@marchbold is your extension using look up tables as well? We started using it and we have devices that have jumped to the wrong scale now (we use it to calculate the state of starling's scale). 
For instance, iPad A16 is tiny af. iPhone 17 as well.

**Comment by marchbold** [comment]
Yes, I have updated them all in my new Display extension which I'm asking all devs to switch to. You can find the latest beta here: https://github.com/distriqt/ANE-Application/issues/313
